Godeps/_workspace/src/github.com/onsi/ginkgo/ginkgo/bootstrap_command.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"go/build"
 
-	"github.com/onsi/ginkgo/ginkgo/nodot"
+	"github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/ginkgo/ginkgo/nodot"
```

Comment:
I don't think you are supposed to do this. The `godep` prefix will make it so that the Godep folder is the first being looked into.

Also it would be quite a pain when running `godep update my_package/path` to have to update this line again. And we would have to make sure that any new import is actually going to the `Godeps` folder.